convert.go

```diff
@@ -50,7 +50,7 @@ func Translate(c *Config) error {
 	defer bfd.Flush()
 
 	// Write the header. This makes e.g. Github ignore diffs in generated files.
-	if _, err = fmt.Fprintf(bfd, "// Package %s Code generated by go-bindata. (@generated) DO NOT EDIT.\n", c.Package); err != nil {
+	if _, err = fmt.Fprintf(bfd, "// Code generated by go-bindata. Package %s. (@generated) DO NOT EDIT.\n", c.Package); err != nil {
```

Comment:
:pick: Is the `Package %s` there to suppress some kind of `golint` error? Does it really need to be in this header?

Also, what is the `(@generated)` text for?

Reply:
The "Package %s" as the line beginning is meant to conform to the package comment standard:
```Golang
// Package mypkg is ...
package mypkg
```

I suggest splitting the original line in three lines:
```Golang
// Code generated ... DO NOT EDIT.

// Package xxxxx
```

This would match both generated code headline (https://golang.org/s/generatedcode) and package documentation standard (https://blog.golang.org/godoc-documenting-go-code)

Reply:
It seems weird to me that go-bindata would output the following program:

``` go
// Code generated by go-bindata. DO NOT EDIT.

// Package mypkg.
package mypkg
```

I could understand if there were some useful documentation that gets added, but that doesn’t happen. And with the code-generation header, I’m pretty sure that the lack of package documentation will be ignored by most tools.

But I don’t actually object to the spurious comment. :man_shrugging:

Reply:
The "// Package mypkg" was left in my suggestion because it was present originally (mixed with the code-generation header), but omitting it would probably be fine